scripts/cleanup-crc-pv.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
i think it will be worth adding `set -ex` here

Reply:
Yes. That's better. Added it in the latest version.